scene/3d/camera.cpp

```diff
@@ -333,7 +333,7 @@ Vector3 Camera::project_ray_origin(const Point2 &p_pos) const {
 bool Camera::is_position_behind(const Vector3 &p_pos) const {
 
 	Transform t = get_global_transform();
-	Vector3 eyedir = -get_global_transform().basis.get_axis(2).normalized();
+	Vector3 eyedir = -t.basis.get_axis(2).normalized();
 	return eyedir.dot(p_pos) < (eyedir.dot(t.origin) + near);
```

Comment:
I do wonder whether this could be reshaped as `eyedir.dot(p_pos - t.origin) < near` to avoid one of the two dot productions with introducing floating-point accuracy issues?

Reply:
I tried this out locally and could not find any ill effects for my personal use case, culling multi mesh instances behind the main camera with a near plane of 0.05, so I added this as a separate commit so it can be easily removed in case it does imply accuracy issues.